envs/environment_py38_iris30.yml

```diff
@@ -21,11 +21,7 @@ dependencies:
   - scipy=1.6
   - sigtools
   - sphinx
-  - statsmodels
   # Optional
-  - numba
-  - pysteps=1.4.1
-  - timezonefinder=4.1.0
```

Comment:
Why are these removed?

Reply:
There is an interest in representing different things with these different environments with this environment being slightly slimmer. This is the approach chosen by @tjtg in https://github.com/gavinevans/improver/pull/13.